src/stakingOptions.ts

```diff
@@ -238,6 +238,49 @@ export class StakingOptions {
     );
   }
 
+  /**
+   * Creates a transaction for config that includes setup instructions
```

Comment:
Is this the prefered solution or do we just want to add a prerequisite ix here if the quote account doesn't exist? https://github.com/Dual-Finance/dual-labs-dip/blob/2644636394b9324ae5b4d929be3acd194d3add44/app/src/utils/studio_contract.ts#L77

Reply:
Do we need this one? With config v3 for reversible, the project quote account only needs to exist when the project does a withdraw

Reply:
should probably take that out of the program, but can do that later